packages/ui/src/components/atoms/Image/Image.tsx

```diff
@@ -3,16 +3,21 @@ import { BaseImage } from '@/components/atoms/Image/BaseImage';
 import { ComponentProps, ElementRef, forwardRef, Suspense } from 'react';
 import { ErrorBoundary } from 'react-error-boundary';
 import { Skeleton } from '@/components';
+import { ctw } from '@/common';
 
 export const Image = forwardRef<ElementRef<'img'>, ComponentProps<typeof BaseImage>>(
   ({ width, height, ...props }, ref) => {
     return (
       <ErrorBoundary
         fallback={
           <figure
-            className={`border-destructive flex flex-col items-center justify-center space-y-2 rounded-md border p-1`}
-            style={{ width, height }}
             aria-live={`polite`}
+            {...props}
+            className={ctw(
+              `border-destructive flex flex-col items-center justify-center space-y-2 rounded-md border p-1`,
+              props?.className,
+            )}
+            style={{ width, height, ...props?.style }}
```

Comment:
Wouldn't it be better to extract `style` from props, the same way we do for `width` and `height`?

Reply:
Difference is `width` and `height` are props I added myself to make them required as opposed to the original optional. Up to you if you still think there's value in doing so.